R/sidebar.R

```diff
@@ -14,7 +14,7 @@
 #' * [page_sidebar()] creates a "page-level" sidebar.
 #' * [page_navbar()] creates a multi-panel app with an (optional, page-level)
 #'   sidebar that is shown on every panel.
-#' * `layout_sidebar()` creates a "floating" sidebar layout component which can
+#' * [layout_sidebar()] creates a "floating" sidebar layout component which can
```

Comment:
This would just link back to the same page, so I prefer the current approach of no link